source/extensions/filters/http/gzip/gzip_filter.cc

```diff
@@ -230,9 +231,10 @@ bool GzipFilter::isMinimumContentLength(Http::HeaderMap& headers) const {
   const Http::HeaderEntry* content_length = headers.ContentLength();
   if (content_length) {
     uint64_t length;
-    const bool is_minimum_content_length =
-        StringUtil::atoull(content_length->value().c_str(), length) &&
-        length >= config_->minimumLength();
+    // TODO(dnoe): Make StringUtil::atoull and friends string_view friendly.
```

Comment:
Aha I see this occurred to you also! :) Follow up?

Reply:
Will track this as part of https://github.com/envoyproxy/envoy/issues/6580